src/styles/index.scss

```
@@ -1,3 +1,122 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+.card {
+  width: 200px;
   margin: 0;
+  font-family: Roboto,sans-serif;
+  font-weight: 400;
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    margin-left: 3px;
+    max-width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+  }
+
+  &__discription {
+   color: #060B35;
+   padding-top: 40px;
+   font-size: 12px;
+   font-weight: 500;
+   line-height: 18px;
+   letter-spacing: 0;
+   height: 36px;
+  }
+
+  &__prodact {
+    height: 14px;
+    color: #616070;
+    font-size: 10px;
+    padding-top: 4px;
+    margin-inline-start: 0;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    padding-top: 16px;
+
+    &-stars {
+      display: flex;
+
+      &-star {
+        width: 16px;
+        height: 16px;
+        flex-shrink: 0;
+        margin-right: 4px;
+        background-repeat: no-repeat;
+        background-position: center;
+        background-image: url(../images/star.svg);
+        &:nth-child(-n + 4) {
+          background-image: url(../images/star-active.svg);
+        }
+      }
+    }
+
+    &-count{
+      font-size: 10px;
+      line-height: 14px;
+      color: #060B35;
+      padding-left: 17px;
+    }
+  }
+
+  &__price {
+    justify-content: space-between;
+    display: flex;
+    padding-top: 24px;
+
+    &-text{
+      color:#616070;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-cost {
+        color:#060B35;
+        font-weight:700;
+        font-size:16px;
+        line-height:18px;
+        padding-left: 85px;
+    }
+  }
+
+  &__button {
+    padding-top: 16px;
+
+    &-buy {
+      background-color: #00ACDC;
+      width: 166px;
+      height: 40px;
+      border-radius: 5px;
+      border:none;
+      justify-content:space-around;
+      color: #fff;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-align: center;
+
+      &:hover {
+        border: 1px solid #00ACDC;
+        color:#00ACDC;
```

Create a variable for repeated color and use it everywhere
```suggestion
        color: #00ACDC;
```